Resources/doc/switcher.md

```diff
@@ -111,3 +111,24 @@ It is also possible to override the default template used in the switcher by pas
  ``` html
 {{ locale_switcher(null, null, 'AcmeDemoBundle:Foo:bar.html.twig') }}
 ```
+
+The template receive as variable a collection of locales available to switch on, these locales contains some
```

Comment:
The template receive*s*

and i would do a `.` rather than `,` after "to switch on" to make it more readable.